<ui version="4.0" >
 <class>PrintOptionsPage</class>
 <widget class="QWidget" name="PrintOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Image Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Image position:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="mPosition" >
       <item>
        <property name="text" >
         <string>Top-Left</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Top-Central</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Top-Right</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Central-Left</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Central</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Central-Right</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Bottom-Left</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Bottom-Central</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Bottom-Right</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>101</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Scaling</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="mNoScale" >
        <property name="text" >
         <string>&amp;No scaling</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mScaleToPage" >
        <property name="text" >
         <string>&amp;Fit image to page</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="mEnlargeSmallerImages" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Enlarge smaller images</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>240</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="mScaleTo" >
        <property name="text" >
         <string>&amp;Scale to:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="mWidth" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="value" >
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>x</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="mHeight" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="value" >
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="mUnit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="currentIndex" >
           <number>1</number>
          </property>
          <item>
           <property name="text" >
            <string>Millimeters</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Centimeters</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Inches</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mKeepRatio" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Keep ratio</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kcombobox.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >kcombobox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>mUnit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>mKeepRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleToPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>mEnlargeSmallerImages</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>mWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScaleTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>mHeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
